src/graphics.rs

```diff
@@ -771,6 +772,8 @@ impl TransformStack {
     }
 }
 
+/// The main graphics struct that combines a bunch of mq types and the
+/// model view matrix to create a basic window that can be drawn into
```

Comment:
I would say this is the graphics context type rather than a graphics struct that creates a basic window; this type is not responsible for window creation, which is done by functionality of the `events` module that handles event loops and window shit.